### PR TITLE
Update setting-the-default-dpi-awareness-for-a-process.md

### PR DESCRIPTION
**Pull request by justanotheranonymoususer**
Just burned about on hour on this broken manifest :(
Also make more consistent with:
https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests

### CODE REVIEW

**Comment by PRMerger8** [comment]
@justanotheranonymoususer : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@mcleanbyron : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mcleanbyron** [comment]
Thanks for your contribution. I edited the manifest example to move the namespaces into each of the DPI-related elements for brevity.

File: desktop-src/hidpi/setting-the-default-dpi-awareness-for-a-process.md [hunk 1]
```diff
@@ -70,13 +70,13 @@ The table below shows how to specify different process-default DPI awareness mod
...
-    <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
-    <application xmlns="urn:schemas-microsoft-com:asm.v3">
-        <windowsSettings>
-            <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
-            <dpiAware>true</dpiAware>
-        </windowsSettings>
-    </application>
+    <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+      <asmv3:application>
+        <asmv3:windowsSettings >
```
Comment (justanotheranonymoususer) [inline desktop-src/hidpi/setting-the-default-dpi-awareness-for-a-process.md:75]: Cool, but you left this annoying space here.
Also, XML sucks (not your fault of course)

**Pull request merged**